.github/workflows/issue-stale-pr.yaml

```diff
@@ -12,7 +12,7 @@ jobs:
   stale:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/stale@main
+      - uses: actions/stale@v9
```

Comment:
Why this is a versioned tag instead of main?

Reply:
It's a good practice to not use the main branch, so I suggest using the latest stable